src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Looking to the diff again, it seems that NOTIFICATION_WAIT_MODE has not been touched by this merge... What has been added has been the comment preamble to the define. So, asking again "Is there any real change in this file?" is seems that there isn't any change (appart from comments, of course).